### ISSUE TITLE
Section 3.4.2, page 85, keeping the many side sorted, paragraph 2, last line

### ISSUE THREAD

**Issue by MudassarBashir** [GrailsInAction/graina2#123]
There is a reference made which reads as:  

...(which you used in our Profile example in listing 3.10).  

But listing 3.10 doesn't seem to match what this reference is hinting to. Apology in advance in case I've misunderstood something.

**Comment by pledbrook** [comment]
You're correct, that's a bad reference and should be removed. Thanks for the report!
